hammerblade/torch/tests/hypothesis_test_util.py

```diff
@@ -25,14 +25,14 @@ def dims(min_value=1, max_value=5):
 def elements_of_type(dtype=np.float32, filter_=None):
     elems = None
     if dtype is np.float32:
-        elems = st.floats(min_value=-128.0, max_value=128.0, width=32)
+        elems = st.floats(min_value=-2.0, max_value=2.0, width=32)
```

Comment:
I think larger values are necessary for testing kernels with exponentials (like softmax)... may be we could disable them temporarily.

Reply:
That is what I'm worried about as well ... I have no idea what is the best way of handling this

Reply:
maybe xfail on travis? http://doc.pytest.org/en/latest/skipping.html#condition-parameter

Reply:
This sounds good to me.

Actually, just realized adding `rtol` parameter might fix large value mismatches... I'll try that in next the pr.